src/modules/auth/containers/LoginCard.js

```diff
@@ -529,6 +549,7 @@ function PasswordField(props) {
 				onChange={reg.onChange}
 				onBlur={reg.onBlur}
 				innerRef={reg.ref}
+				autoFocus={props.idx == 0}
```

Comment:
@queuing4oranges I think this is not necessary, when you are using the `focusInputField` method you've created